intranet/static/js/dashboard/announcements.js

```diff
@@ -22,22 +22,29 @@ $(document).ready(function() {
 
         var hidden = announcement.hasClass("hidden");
         var action = hidden ? "show" : "hide";
-        $.post("/announcements/" + action + "?" + id, {announcement_id: id}, function() {
-            console.info("Announcement "+id+" "+action);
+
+        $.post("/announcements/" + action + "?" + id, {
+            announcement_id: id
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'announcement_id' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2747525187/issues/source?bid=3212528&fileBranchId=3355681#l27)